### PR TITLE
fix: remove runAlways from changelog

### PR DESCRIPTION
**Pull request by CaiWillis**

### CODE REVIEW

**Review by DorisWongNHS — DISMISSED** [review]

**Review by DorisWongNHS — DISMISSED** [review]

File: application/src/main/java/uk/nhs/hee/tis/revalidation/changelog/TisStatusCorrectionChangeLog.java [hunk 1]
```diff
@@ -40,15 +37,14 @@
 @Slf4j
 public class TisStatusCorrectionChangeLog {
 
-    @ChangeSet(order = "001", id = "insertCorrectTisStatuses", author = "", runAlways = true)
+    @ChangeSet(order = "001", id = "insertCorrectTisStatuses", author = "")
```
Comment (JosephKelly) [inline application/src/main/java/uk/nhs/hee/tis/revalidation/changelog/TisStatusCorrectionChangeLog.java:40]: I'd prefer us to be more definitive about the order that the changesets run in.

**Review by JosephKelly — CHANGES_REQUESTED** [review]

File: application/src/main/java/uk/nhs/hee/tis/revalidation/changelog/TisStatusCorrectionChangeLog.java (hunk 1 above)
Comment (CaiWillis) [reply application/src/main/java/uk/nhs/hee/tis/revalidation/changelog/TisStatusCorrectionChangeLog.java:40]: Do you mean by adding an order to the ChangeLog annotation?

Comment (JosephKelly) [reply application/src/main/java/uk/nhs/hee/tis/revalidation/changelog/TisStatusCorrectionChangeLog.java:40]: Yep.  From a bit of digging it looks like the order is only effective within a single `@Changlog` so ensuring ordering would require a larger refactor.  We should take a look at a larger refactor (separately) as there are other Mongock best practices that we should consider.  https://mongock.io/v5/migration/index.html#best-practices

Comment (CaiWillis) [reply application/src/main/java/uk/nhs/hee/tis/revalidation/changelog/TisStatusCorrectionChangeLog.java:40]: That's a good read, thanks!

**Comment by DorisWongNHS** [comment]
Further to your suggestion @JosephKelly (https://mongock.io/v4/changelogs/index.html#%40changelog), I have added the order for each changeLog. It ran successfully on my local. Thanks very much for the suggestion.

**Review by JosephKelly — APPROVED** [review]
This change itself is a good one.  Comment is about a follow up.

**Pull request merged**